wallet-new/src/Cardano/Wallet/Kernel/DB/Sqlite.hs

```diff
@@ -280,9 +260,9 @@ address :: DataType SqliteDataTypeSyntax Core.Address
 address = DataType (varCharType Nothing Nothing)
 
 -- | 'DataType' declaration to convince @Beam@ treating 'Core.Timestamp'(s) as
--- varchars of arbitrary length.
+-- a SQL @INTEGER@.
 timestamp :: DataType SqliteDataTypeSyntax Core.Timestamp
-timestamp = DataType (varCharType Nothing Nothing)
+timestamp = DataType intType
```

Comment:
hmmm, so that may actually give us some truncation. we might want to use the [`UNSIGNED BIG INT`](https://www.sqlite.org/datatype3.html#affinity_name_examples) type instead, which is an 8 byte unsigned int (eg a `Word64)`. 

I mean, it's *fine*, if we really care about the difference between overflow in 52,000 CE and 26,000 CE, we should be using a string :laughing: